sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -771,24 +771,23 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
           else
           {
             unused(context);
-            throw Azure::Storage::StorageException::CreateFromResponse(std::move(responsePtr));
+            throw Storage::StorageException::CreateFromResponse(std::move(responsePtr));
```

Comment:
Do we need `Storage::` here?

Reply:
No we don't, but can't be bother to change that.

Reply:
whatever